go_lib/dependency/dependency.go

```diff
@@ -257,6 +263,27 @@ func (mdc *mockedDependencyContainer) SetK8sVersion(ver k8s.FakeClusterVersion)
 	mdc.K8sClient = cli
 }
 
+// CreateDynamicFromYAML create dynamic object from yaml string
+// you can create multiply objects of the same kind by separating yaml with '---'
+func (mdc *mockedDependencyContainer) CreateDynamicFromYAML(gvr schema.GroupVersionResource, data string) error {
+	arr := strings.Split(data, "---")
```

Comment:
This split pattern is error-prone. There is a 'ListFromYamlDocs' in [kube-client](https://github.com/flant/kube-client/blob/master/manifest/manifest_list.go#L5) that is used for monitoring Helm resources in addon-operator.

Reply:
Well. It was for tests only but you are right. I have fixed it